apps/examples/st_things/st_things_sample_main.c

```diff
@@ -0,0 +1,67 @@
+/* ****************************************************************
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+*
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+******************************************************************/
+
+#include <tinyara/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <unistd.h>
+#include <inttypes.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <errno.h>
+#include <st_things/st_things.h>
+#include "st_things_sample.h"
+#include <time.h>
```

Comment:
In here, you can remove unnecessay including headers like string.h, stdbool.h fcntl.h and so on. Please remove them.

Reply:
modified

Reply:
Please let me know the reason for inttypes.h, stat.h, errno.h and st_things.h

Reply:
removed